samples/web/content/apprtc/js/call.js

```diff
@@ -191,11 +192,12 @@ Call.prototype.connectToRoom_ = function(roomId, mediaPromise, turnPromise) {
     // and have media and TURN. Since we send candidates as soon as the peer
     // connection generates them we need to wait for the signaling channel to be
     // ready.
-    Promise.all([turnPromise, mediaPromise]).then(function() {
-      this.startSignaling_();
-    }.bind(this)).catch(function(error) {
-      this.onError_('Failed to start signaling: ' + error.message);
-    }.bind(this));
+    Promise.all([this.getTurnServersPromise_, this.getMediaPromise_])
```

Comment:
would it be easier to just call maybeGetMedia here? getMediaPromise_ doesn't seem to be used elsewhere.